packages/special-pages/pages/duckplayer/src/js/index.js

```diff
@@ -86,10 +87,42 @@ const VideoPlayer = {
      * Sets up the video player:
      * 1. Fetches the video id
      * 2. If the video id is correctly formatted, it loads the YouTube video in the iframe, otherwise displays an error message
+     * @param {object} opts
+     * @param {string} opts.base
      */
-    init: () => {
+    init: (opts) => {
         VideoPlayer.loadVideoById()
         VideoPlayer.setTabTitle()
+        VideoPlayer.setClickListener(opts.base)
+    },
+
+    /**
+     * In certain circumstances, we may want to intercept
+     * clicks within the iframe - for example when showing a video
+     * that cannot be played in the embed
+     *
+     * @param {string} urlBase - macos/windows current use a different base URL
+     */
+    setClickListener: (urlBase) => {
+        VideoPlayer.onIframeLoaded(() => {
+            const iframe = VideoPlayer.iframe()
+            iframe.contentDocument?.addEventListener('click', (e) => {
```

Comment:
What about touch etc?

Reply:
Oh that's a good point - I'll fire this up on my windows machine and verify 👍🏻